rasa/core/config.py

```diff
@@ -25,10 +30,6 @@
 
 import rasa.shared.utils.io
 import rasa.utils.io
-from rasa.core.policies.mapping_policy import MappingPolicy
-from rasa.core.policies.rule_policy import RulePolicy
-from rasa.core.policies.fallback import FallbackPolicy
-from rasa.core.policies.two_stage_fallback import TwoStageFallbackPolicy
 from rasa.nlu.classifiers.fallback_classifier import FallbackClassifier
```

Comment:
optional: one more request: Could you please also replace this one with a constant? This gives us the massive boost for `test_rasa_data` from 84 seconds to 38 seconds.